config/loki/values.yaml

```diff
@@ -90,6 +90,12 @@ loki:
 
 tableManager:
   enabled: true
+  extraArgs:
```

Comment:
why do we need s3 for tableManager?

Reply:
this is needed, otherwise it will throw AWS credential error